src/StoreKeeper/WooCommerce/B2C/Models/AbstractModel.php

```diff
@@ -63,11 +63,19 @@ public static function ensureTable(): bool
             }
 
             return false;
+        } elseif (static::isTableOutdated()) {
+            static::alterTable();
+            static::setTableVersion();
         }
 
         return true;
     }
 
+    protected static function isTableOutdated(): bool
+    {
+        return static::getTableVersion() !== static::TABLE_VERSION;
```

Comment:
Outdated means older not different use version compare

Reply:
Changed to `version_compare()`